scripts/deploy.ts

```diff
@@ -110,6 +178,90 @@ async function main() {
   )
 }
 
+const legacyContractAddress = '0x9e7A15E77e5E4f536b8215aaF778e786005D0f8d'
+
+async function downloadData(
+  factory: OBSSStorage__factory,
+  provider: JsonRpcProvider,
+  signer: Signer
+) {
+  const legacyContract = factory
+    .attach(legacyContractAddress)
+    .connect(provider)
+    .connect(signer)
+
+  console.log(legacyContract.address)
+  const totalFeeds = await legacyContract.lastFeedId()
+  console.log(`Total feeds count: ${totalFeeds.toNumber()}`)
+  const legacyPosts: OBSSStorage.LegacyPostStruct[] = []
+  const legacyReactions: OBSSStorage.LegacyReactionStruct[] = []
+  for (let i = 0; i < 10; i++) {
+    const postsInFeed = await legacyContract.lastFeedPostIds(i)
+    if (postsInFeed.toNumber() === 0) continue
+
+    const feedPosts = await legacyContract.getFeedPosts(
+      i,
+      0,
+      postsInFeed.toNumber()
+    )
+    console.log(`Feed ID: ${i}`)
+    feedPosts.forEach(async (post: OBSSStorage.PostStructOutput, j) => {
+      legacyPosts.push({
+        post: {
```

Comment:
Maybe we can do `post: { ...post, timestamp: post.timestamp.toNumber(), commentsFeedId: 0}`